apps/system/js/browser_context_menu.js

```diff
@@ -171,6 +193,31 @@
       }));
     },
 
+    pinUrl: function(url) {
+      var data = {
+        type: 'url',
+        url: url,
+        iconable: false,
+        title: this.app.title,
+        theme: this.app.themeColor
```

Comment:
Is the string "theme" used elswhere already? Seems like it should be "theme-color"?